.github/workflows/release-tag-versions.yaml

```diff
@@ -31,10 +32,13 @@ on:
         description: "The GitHub token for committing the changes."
         required: true
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}-release-version
+  cancel-in-progress: true
+
```

Comment:
We do not want workflows that create tags or releases to every be canceled while they are running